Owin.Security.Providers/LinkedIn/Provider/LinkedInAuthenticatedContext.cs

```diff
@@ -36,8 +36,9 @@ public LinkedInAuthenticatedContext(IOwinContext context, JObject user, string a
 
             Id = TryGetValue(user, "id");
             Name = TryGetValue(user, "formattedName");
+            FamilyName = TryGetValue(user, "lastName");
+            GivenName = TryGetValue(user, "firstName");
             Link = TryGetValue(user, "publicProfileUrl");
-            UserName = TryGetValue(user, "formattedName").Replace(" ", "");
```

Comment:
Please restore this as well. It makes no sense putting back the `UserName` property if its value is always going to be null :) Then it still breaks people's code.